dusty/commands/setup.py

```diff
@@ -109,6 +117,8 @@ def setup_dusty_config(mac_username=None, specs_repo=None, nginx_includes_dir=No
     else:
         boot2docker_vm_memory = _get_boot2docker_vm_size()
 
+    _check_ssh_add()
```

Comment:
The problem with moving it is that we fail to update your git repos without this value being set.  Which means we get this error: 
```
ERROR: Git command failed. If you are trying to access a remote repository over SSH (e.g. GitHub), please make sure you have added your SSH key to the SSH agent using: ssh-add -K <SSH key filepath>
ERROR: 'Error when fetching: fatal: Could not read from remote repository.' returned with exit code 2
```